components/shared/modal/ErrorModal.js

```diff
@@ -19,7 +18,12 @@ const ErrorModal = ({ isVisible, closeModal }) => {
         <TouchableOpacity onPress={closeModal}>
           <Text>{$t('error.cancel')}</Text>
         </TouchableOpacity>
-        <TouchableOpacity onPress={() => NavigationService.navigate('AuthLoading')}>
+        <TouchableOpacity
+          onPress={() => {
+            closeModal();
+            Updates.reload();
```

Comment:
Can you move this to a function please? So we have a more clear code :)

Reply:
yes, done